prometheus/summary.go

```diff
@@ -54,6 +54,9 @@ type Summary interface {
 }
 
 // DefObjectives are the default Summary quantile values.
+//
+// Deprecated: DefObjectives will be removed in v0.10. The default will be a
```

Comment:
Sure, we can leave them in place, but I'd still rename then. Let's decide about that later (when we actually change the default behavior). For now, nothing changes. I have changed the comment to not suggest that they will be removed (rather: not used as default anymore).